modules/framework/src/weaver/framework/DogFood.scala

```diff
@@ -28,8 +28,8 @@ abstract class DogFood[F[_]](
   // for some time before getting the logs back. On JVM platform
   // we do not need to wait, since the suite will run synchronously
   private val patience: Option[FiniteDuration] = PlatformCompat.platform match {
-    case JS  => 2.seconds.some
-    case JVM => none
+    case JS | Native => 2.seconds.some
```

Comment:
For my own understanding - the comment above talks about SJS being asynchronous, hence the need for waiting.

Do we still need it for Native, which will also execute synchronously? Or is it something else?

Reply:
[aa7296e](https://github.com/disneystreaming/weaver-test/pull/568/commits/aa7296e2be5a9ba2cd6ccad7cda27c5b76c76b81)